templates/template-for-tasks.rst

```diff
@@ -0,0 +1,171 @@
+:orphan: true
+	 
+.. Based on:
+   https://dmtn-030.lsst.io/v/DM-7096/index.html#task-topic-type, with
+   learnings from the 4 sfp pages built in branch DM-8717
+
+Please fill in the below sections with the contents as described in
+the reST comments, removing the comments as you do so, and using the
+links to locations in the `Guidance Document
+<writing-task-topics.html>`_ if needed.
+
+
+########################
+Structure of ExampleTask 
+########################
+
+.. Fill in introductory material here - this section needs the following:
+
+.. - Summary/context (1-2 sentences).
+
+.. - Concise summary of logic/algorithm in a paragaph and/or bullet list.
+
+.. - A sentence about each step, which can be either:
+
+..  a) A retargetable subtask
+
+..  b) A method within a task.
+
+`Guidance for the Introduction Section  <writing-task-topics.html#task-topics-intro>`_ .
+
+
+.. - Insert Module Membership here:
```

Comment:
Ok, I inserted the above (leaving the link in currently to that section in the Guidance doc, but modifying wording there some), and removed extraneous blank lines everywhere (easier to *read* with blank lines in, for me, but more of a pain to go scroll through and delete them.)